reference/5.1/Microsoft.PowerShell.Core/About/about_Arrays.md

```diff
@@ -26,47 +26,47 @@ To create and initialize an array, assign multiple values to a variable. The
 values stored in the array are delimited with a comma and separated from the
 variable name by the assignment operator (=).
 
-For example, to create an array named $A that contains the seven numeric (int)
+For example, to create an array named `$A` that contains the seven numeric (int)
 values of 22, 5, 10, 8, 12, 9, and 80, type:
 
 ```powershell
 $A = 22,5,10,8,12,9,80
 ```
 
-You can also create and initialize an array by using the range operator (..).
-For example, to create and initialize an array named "$B" that contains the
-values 5 through 8, type:
+You can also create and initialize an array by using the range operator (`..`).
+The following example creates an array named containing the values 5 through 8.
 
 ```powershell
 $B = 5..8
 ```
 
-As a result, $B contains four values: 5, 6, 7, and 8.
+As a result, `$B` contains four values: 5, 6, 7, and 8.
 
-When no data type is specified, PowerShell creates each array as an
-object array (type: System.Object[]). To determine the data type of an array,
-use the GetType() method. For example, to determine the data type of the $a
-array, type:
+When no data type is specified, PowerShell creates each array as an object
+array (**System.Object[]**). To determine the data type of an array, use the
+GetType() method. For example, to determine the data type of the `$a` array,
```

Comment:
Method should be bold or fenced. Also, if this is the same array as farther above, do you want to use the same case?

Reply:
Fixed both